services/web/server/src/simcore_service_webserver/folders/_workspaces_handlers.py

```diff
@@ -0,0 +1,79 @@
+import functools
+import logging
+from typing import Annotated
+
+from aiohttp import web
+from models_library.folders import FolderID
+from models_library.utils.common_validators import null_or_none_str_to_none_validator
+from models_library.workspaces import WorkspaceID
+from pydantic import BaseModel, BeforeValidator, ConfigDict, Field
+from servicelib.aiohttp import status
+from servicelib.aiohttp.requests_validation import parse_request_path_parameters_as
+from servicelib.aiohttp.typing_extension import Handler
+
+from .._meta import api_version_prefix as VTAG
+from ..folders.errors import FolderAccessForbiddenError, FolderNotFoundError
+from ..login.decorators import login_required
+from ..projects.exceptions import ProjectInvalidRightsError, ProjectNotFoundError
+from ..security.decorators import permission_required
+from ..workspaces.errors import WorkspaceAccessForbiddenError, WorkspaceNotFoundError
+from . import _workspaces_api
+from ._models import FoldersRequestContext
+
+_logger = logging.getLogger(__name__)
+
+
+def _handle_folders_workspaces_exceptions(handler: Handler):
```

Comment:
No,  there is not need for this anymore. Please use the new approach. See how `_trash_handlers` and `folders_handlers` get the exception handling decorator from `_exceptions_handlers`.

Reply:
In that case should we maybe create the models based on the subresource, for example `_workspaces_models` ? -> But to be honest I do not see a big need for that, I am ok that the model lives in the _resource_handlers as the model is connected directly to the handler.